src/types/state.ts

```diff
@@ -39,6 +52,8 @@ export type SelectedRevisionsState = {
   revisions: Revision[];
 };
 
+export type CompareResultsState = CompareResultsItem[];
```

Comment:
Don't you need to add `compareResults` below to the `State` too?

Looking at https://github.com/mozilla/perfcompare/pull/149/files#diff-c1cf636d61418b4439ce697bd7f4249145469dacd55453c8832c045f1a99f5cbR9-R14 it seems that some other properties are missing too. Is that expected that TypeScript doesn't error about that?

Reply:
I originally was using this `State` type, but at some point added `RootState` that infers the type: https://github.com/mozilla/perfcompare/blob/master/src/common/store.ts#L21

I will file an issue to remove `State` and replace with `RootState`

my guess on why it isn't erroring is that the only place `State` is used, it is not accesing the missing state types: https://github.com/mozilla/perfcompare/pull/149/files#diff-f4aeacfb95826ff4c36316d81101147c633ca4a49655531c49c6dff7d4767970R33